hdr/sqlite_modern_cpp.h

```diff
@@ -787,6 +795,24 @@ namespace sqlite {
 		val = i;
 	}
 
+	 /// apply a member function
+	 inline database_binder& operator <<(database_binder& db, void (database_binder::* pf)(void)) {
+	   (db.*pf)();
+	   return db;
+	 }
+
+	 /// apply a function pointer
+	 inline database_binder& operator<< (database_binder& db, void (*fun)(database_binder*)) {
+	   fun(&db);
+	   return db;
+	 }
+
+	 /// apply a lambda
+	 inline database_binder& operator<< (database_binder& db, std::function<void(database_binder&)>f) {
+	   f(db);
+	   return db;
+	 }
+
```

Comment:
Can we use our `function_traits` instead of `std::function` here to avoid type-erasure?
What is the use-case for calling this with a lambda?